account.go

```diff
@@ -0,0 +1,107 @@
+package archethic
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"strings"
+)
+
+func NewKeychainTransaction(seed []byte, authorizedPublicKeys [][]byte) TransactionBuilder {
+	keychain := NewKeychain(seed)
+	keychain.AddService("uco", "m/650'/0/0", ED25519, SHA256)
+
+	aesKey := make([]byte, 32)
+	rand.Read(aesKey)
+
+	authorizedKeys := make([]AuthorizedKey, len(authorizedPublicKeys))
+	for i, key := range authorizedPublicKeys {
+		authorizedKeys[i] = AuthorizedKey{
+			publicKey:          key,
+			encryptedSecretKey: EcEncrypt(aesKey, key),
+		}
+	}
+
+	tx := TransactionBuilder{}
+	tx.SetType(KeychainType)
+	keychain.ToDID()
```

Comment:
What is the purpose of this line ? Maybe a miss ;)

Reply:
eheh well spotted! 😅